src/helpers/discord_utils.ts

```diff
@@ -258,14 +258,15 @@ export async function sendOptionsMessage(messageContext: MessageContext, guildPr
         {
             title: updatedOption === null ? "Options" : `${updatedOption.option} ${updatedOption.reset ? "reset" : "updated"}`,
             description:
-                `Now playing the ${optionStrings[GameOption.LIMIT]} most popular songs out of the __${totalSongs.countBeforeLimit}__ by ${guildPreference.isGroupsMode() ? `${optionStrings[GameOption.GROUPS]} (${optionStrings[GameOption.SUBUNIT_PREFERENCE]})` : `${optionStrings[GameOption.GENDER]} ${optionStrings[GameOption.ARTIST_TYPE]}`}\
+                `-Now playing the ${optionStrings[GameOption.LIMIT]} most popular songs out of __${totalSongs.countBeforeLimit}__ by ${guildPreference.isGroupsMode() ? `${optionStrings[GameOption.GROUPS]} (${optionStrings[GameOption.SUBUNIT_PREFERENCE]})` : `${optionStrings[GameOption.GENDER]} ${optionStrings[GameOption.ARTIST_TYPE]}`}\
```

Comment:
You should be able to remove the `\` at the end of every line as well as the `\n` at the start of every line, since template literals (strings using ` instead of ") support multi-line strings.